### PR TITLE
Fix attach data links, add warning regarding VEC

### PR DESCRIPTION
**Pull request by rymorale**
Also shorten Target attach data section names.

### CODE REVIEW

File: resources/user-guides/attach-data.md [hunk 1]
```diff
@@ -60,20 +60,20 @@ When defining a payload for the attach data action, the payload must match the f
...
 The following sample shows how to _attach data_ to all outgoing `TrackAction` Analytics network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
 
-1. [Create a new **Rule**](attach-data.md#analytics-create-rule)
-2. [Select the **Event** you want to trigger the rule](attach-data.md#analytics-select-an-event)
-3. [Select the **Action** to attach data and define your payload](attach-data.md#analytics-define-the-action)
-4. [Save and rebuild the property](attach-data.md#analytics-save-the-rule-and-rebuild-your-property)
+1. [Create a new **Rule**](#analytics-create-rule)
+2. [Select the **Event** you want to trigger the rule](#analytics-select-an-event)
+3. [Select the **Action** to attach data and define your payload](#analytics-define-the-action)
+4. [Save and rebuild the property](#analytics-save-the-rule-and-rebuild-your-property)
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: please add a period at the end of each step, like this:
"4. [Save and rebuild the property](#analytics-save-the-rule-and-rebuild-your-property)."

File: resources/user-guides/attach-data.md [hunk 2]
```diff
@@ -110,25 +110,30 @@ After you complete your configuration, verify that your rule looks like the foll
 2. Rebuild your Launch property and deploy it to the correct Environment.
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please replace "Launch property" with "Experience Platform Launch property"

File: resources/user-guides/attach-data.md [hunk 3]
```diff
@@ -110,25 +110,30 @@ After you complete your configuration, verify that your rule looks like the foll
...
 
-{% tab title="Target" %}
-### Attaching additional Target request objects to Target to retrieve location events
+{% tab title="Target" %} <a name="Target-Attach-Data-Tab"></a>
+
+{% hint style="warning" %}
+Attach data events apply to the Target Extension only. They do not apply to the Target VEC Extension. 
+{% endhint %}
+
+### Attaching additional data to Target retrieve location events <a name="Target-Attach-Data-Retrieve-Location"> </a>
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please update the heading to:
"Attach additional data to Target to retrieve location events"

File: resources/user-guides/attach-data.md [hunk 4]
```diff
@@ -110,25 +110,30 @@ After you complete your configuration, verify that your rule looks like the foll
...
 The following sample shows how to _attach data_ to all outgoing `retrieveLocationContent` Target network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
 
-1. [Create a new **Rule**](attach-data.md#target-create-rule)
-2. [Select the **Event** you want to trigger the rule](attach-data.md#target-select-an-event)
-3. [Select the **Action** to attach data and define your payload](attach-data.md#target-define-the-action)
-4. [Save and rebuild the property](attach-data.md#target-save-the-rule-and-rebuild-your-property)
+1. [Create a new **Rule**](#target-create-rule)
+2. [Select the **Event** you want to trigger the rule](#target-select-an-event)
+3. [Select the **Action** to attach data and define your payload](#target-define-the-action)
+4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property)
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please add a period to the end of each step, like this:
"4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property)."

File: resources/user-guides/attach-data.md [hunk 5]
```diff
@@ -181,24 +186,24 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
-### Attaching additional Target request objects to Target to prefetch content events
+### Attaching additional data Target to prefetch content events<a name="Target-Attach-Data-Prefetch-Content"></a>
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please update the heading as follows:
"Attach additional data to Target to prefetch content events"

File: resources/user-guides/attach-data.md [hunk 6]
```diff
@@ -181,24 +186,24 @@ After you complete your configuration, verify that your rule looks like the foll
...
 The following sample shows how to add a custom mbox to prefetch in all outgoing `prefetchContent` Target network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
 
-1. [Create a new **Rule**](attach-data.md#target-create-rule-prefetch)
-2. [Select the **Event** you want to trigger the rule](attach-data.md#target-select-an-event-prefetch)
-3. [Select the **Action** to attach data and define your payload](attach-data.md#target-define-the-action-prefetch)
-4. [Save and rebuild the property](attach-data.md#target-save-the-rule-and-rebuild-your-property-prefetch)
+1. [Create a new **Rule**](#target-create-rule-prefetch)
+2. [Select the **Event** you want to trigger the rule](#target-select-an-event-prefetch)
+3. [Select the **Action** to attach data and define your payload](#target-define-the-action-prefetch)
+4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-prefetch)
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please add a period to the end of each step, like this:
"4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-prefetch)."

File: resources/user-guides/attach-data.md [hunk 7]
```diff
@@ -251,24 +256,24 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
-### Attaching additional Target parameters to Target location-clicked events
+### Attaching additional data to Target location-clicked events<a name="Target-Attach-Data-Location-Clicked"></a>
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please update the heading to:
"Attach additional data to Target location-clicked events"

File: resources/user-guides/attach-data.md [hunk 8]
```diff
@@ -251,24 +256,24 @@ After you complete your configuration, verify that your rule looks like the foll
...
 The following sample shows how to add additional mbox and profile parameters in all outgoing `locationClicked` Target network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
 
-1. [Create a new **Rule**](attach-data.md#target-create-rule-clicked)
-2. [Select the **Event** you want to trigger the rule](attach-data.md#target-select-an-event-clicked)
-3. [Select the **Action** to attach data and define your payload](attach-data.md#target-define-the-action-clicked)
-4. [Save and rebuild the property](attach-data.md#target-save-the-rule-and-rebuild-your-property-clicked)
+1. [Create a new **Rule**](#target-create-rule-clicked)
+2. [Select the **Event** you want to trigger the rule](#target-select-an-event-clicked)
+3. [Select the **Action** to attach data and define your payload](#target-define-the-action-clicked)
+4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-clicked)
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please add a period to the end of each step, like this:
"4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-clicked)."

File: resources/user-guides/attach-data.md [hunk 9]
```diff
@@ -308,24 +313,24 @@ After you complete your configuration, verify that your rule looks like the foll
 1. Click **Save**
 2. Rebuild your Launch property and deploy it to the correct Environment.
 
-### Attaching additional Target parameters to Target location-displayed events
+### Attaching additional data to Target location-displayed events<a name="Target-Attach-Data-Location-Displayed"></a>
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please update the heading as follows:
"Attach additional data to Target location-displayed events"

File: resources/user-guides/attach-data.md [hunk 10]
```diff
@@ -308,24 +313,24 @@ After you complete your configuration, verify that your rule looks like the foll
...
 The following sample shows how to add additional mbox and profile parameters in all outgoing `locationDisplayed` Target network requests. To create this type of rule, select your property in Experience Platform Launch and complete the following steps:
 
-1. [Create a new **Rule**](attach-data.md#target-create-rule-displayed)
-2. [Select the **Event** you want to trigger the rule](attach-data.md#target-select-an-event-displayed)
-3. [Select the **Action** to attach data and define your payload](attach-data.md#target-define-the-action-displayed)
-4. [Save and rebuild the property](attach-data.md#target-save-the-rule-and-rebuild-your-property-displayed)
+1. [Create a new **Rule**](#target-create-rule-displayed)
+2. [Select the **Event** you want to trigger the rule](#target-select-an-event-displayed)
+3. [Select the **Action** to attach data and define your payload](#target-define-the-action-displayed)
+4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-displayed)
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please add a period to the end of each step, like this:
"4. [Save and rebuild the property](#target-save-the-rule-and-rebuild-your-property-displayed)."

File: resources/user-guides/attach-data.md [hunk 11]
```diff
@@ -110,25 +110,30 @@ After you complete your configuration, verify that your rule looks like the foll
 2. Rebuild your Launch property and deploy it to the correct Environment.
 {% endtab %}
 
-{% tab title="Target" %}
-### Attaching additional Target request objects to Target to retrieve location events
+{% tab title="Target" %} <a name="Target-Attach-Data-Tab"></a>
+
+{% hint style="warning" %}
+Attach data events apply to the Target Extension only. They do not apply to the Target VEC Extension.
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please reword as follows:
"Attach the data events that apply only to the Target extension. The data events do not apply to the Target VEC extension."

File: resources/user-guides/attach-data.md [hunk 12]
```diff
@@ -100,35 +100,40 @@ In the following example, **launches** and **anAddedKey** keys are added to the
...
 ![](../../.gitbook/assets/rulecomplete.png)
 
 1. Click **Save**
-2. Rebuild your Launch property and deploy it to the correct Environment.
+2. Rebuild your Experience Platform Launch property and deploy it to the correct Environment.
 {% endtab %}
 
-{% tab title="Target" %}
-### Attaching additional Target request objects to Target to retrieve location events
+{% tab title="Target" %} <a name="Target-Attach-Data-Tab"></a>
```
Comment (swarna04) [inline resources/user-guides/attach-data.md]: We can just name the link as "Target-Attach-Data"

File: resources/user-guides/attach-data.md [hunk 13]
```diff
@@ -107,23 +107,23 @@ After you complete your configuration, verify that your rule looks like the foll
...
 1. Click **Save**
-2. Rebuild your Launch property and deploy it to the correct Environment.
+2. Rebuild your Experience Platform Launch property and deploy it to the correct Environment.
 {% endtab %}
 
 {% tab title="Target" %} <a name="Target-Attach-Data-Tab"></a>
 
 {% hint style="warning" %}
-Attach data events apply to the Target Extension only. They do not apply to the Target VEC Extension. 
+The Attach Data feature applies **only** to the Target, and not the Target VEC, extension.
```
Comment (rekrishn) [inline resources/user-guides/attach-data.md]: Please remove the comma between "Target" and "and".

**Pull request merged**